Ys cb eedr update version (#34757)
demisto/content: Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/README.md

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| alert_id | The ID of the alert to update. Get the ID from list_alerts command. | Required | 
| status | Workflow status to update. | Optional | 
| comment | Comment to include with the operation. | Optional | 
| closure_reason | The closure reason | Optional |
| determination | Value judgement of whether the alert(s) represent a true or false positive. | Optional |

> | end | The upper bound of the time range Requires start and must be a timestamp after start. | Optional |

Reviewer: ```suggestion
| end | The upper bound of the time range. Requires start and must be a timestamp after start. | Optional |
```

| CarbonBlackEEDR.SearchProcess.results.device_group_id | Number | The ID of the sensor group where the device belongs. | 
| CarbonBlackEEDR.SearchProcess.results.device_name | String | The name of the device. | 
| CarbonBlackEEDR.SearchProcess.results.device_policy_id | Number | The ID of the policy applied to the device. | 
| CarbonBlackEEDR.SearchProcess.results.device_timestamp | Date | The time displayed on the sensor based on the sensor’s clock. The time is an ISO-8601 formatted time string based on the UTC timezone. | 
| CarbonBlackEEDR.SearchProcess.results.enriched | Boolean | True if the process document came from the CBD data stream. | 
| CarbonBlackEEDR.SearchProcess.results.enriched_event_type | String | The CBD enriched event type. | 
| CarbonBlackEEDR.SearchProcess.results.event_type | String | The CBD event type \(valid only for events coming through analytics\). Possible values are: CREATE_PROCESS, DATA_ACCESS, FILE_CREATE, INJECT_CODE, NETWORK, POLICY_ACTION, REGISTRY_ACCESS, and SYSTEM_API_CALL. | 
| CarbonBlackEEDR.SearchProcess.results.filemod_count | Number | The cumulative count of file modifications since process tracking started. | 

> | CarbonBlackEEDR.SearchProcess.results.ingress_time | Date | Unknown | 

Reviewer: What is the description?  Is it really unknown?

| CarbonBlackEEDR.SearchProcess.results.legacy | Boolean | True if the process document came from the legacy data stream \(deprecated, use enriched\). | 
| CarbonBlackEEDR.SearchProcess.results.modload_count | Number | The cumulative count of module loads since process tracking started. | 
| CarbonBlackEEDR.SearchProcess.results.netconn_count | Number | The cumulative count of network connections since process tracking started. | 
| CarbonBlackEEDR.SearchProcess.results.org_id | String | The globally unique organization key. This will most likely be the PSC organization ID \+ PSC environment ID or some other unique token used across environments. | 
| CarbonBlackEEDR.SearchProcess.results.parent_guid | String | The process GUID of the parent process. | 
| CarbonBlackEEDR.SearchProcess.results.parent_pid | Number | The PID of the parent process. | 

> | CarbonBlackEEDR.SearchProcess.results.process_guid | String | Unique ID of the solr document. Appears as process_guid \+ server-side timestamp in epoch ms \(1/1/1970 based\). | 

Reviewer: ```suggestion
| CarbonBlackEEDR.SearchProcess.results.process_guid | String | Unique ID of the Solr document. Appears as process_guid \+ server-side timestamp in epoch ms \(1/1/1970 based\). | 
```

> | CarbonBlackEEDR.SearchProcess.results.process_hash | String | The MD5 and SHA-256 hashes of the process’s main module in a multi-valued field. | 

Reviewer: ```suggestion
| CarbonBlackEEDR.SearchProcess.results.process_hash | String | The MD5 and SHA256 hashes of the process’s main module in a multi-valued field. | 
```

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| CarbonBlackEEDR.Threat.ThreatID | unknown | Threat ID. | 
| CarbonBlackEEDR.Threat.Tags | unknown | Threat tags. | 
### cb-eedr-list-alerts

***
Returns a list of alerts.

#### Base Command

`cb-eedr-list-alerts`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| minimum_severity | Alert minimum severity (In range of 1-10). | Optional | 

> | device_os_version | Device OS version. Supports comma-separated values. | Optional | 

Reviewer: ```suggestion
| device_os_version | A comma-separated list of device operating system versions.  | Optional | 
```

> | policy_id | The policy ID. Supports comma-separated values. | Optional | 
> | alert_tag | Alert tags. Supports comma-separated values. | Optional | 
> | alert_id | Alert ID. Supports comma-separated values. | Optional | 
> | device_username | Device username. Supports comma-separated values. | Optional | 
> | device_id | Device ID. Supports comma-separated values. | Optional | 
> | device_os | Device OS. Supports comma-separated values. | Optional | 
> | process_sha256 | Process SHA256. Supports comma-separated values. | Optional | 
> | policy_name | Policy name. Supports comma-separated values. | Optional | 
> | reputation | Alert reputation. Supports comma-separated values. | Optional | 
> | alert_type | Alert type. Supports comma-separated values. | Optional | 
> | device_name | Device name. Supports comma-separated values. Examples- "C:\\Users\\exmaple_user\\Example" or  "/home/example_user/Example". | Optional | 
> | process_name | Process name. Supports comma-separated values. Examples- "C:\\Users\\exmaple_user\\Example" or  "/home/example_user/Example". | Optional | 

Reviewer: ```suggestion
| policy_id | A comma-separated list of policy IDs. | Optional | 
| alert_tag | A comma-separated list of alert tags. | Optional | 
| alert_id | A comma-separated list of alert IDs.  | Optional | 
| device_username | A comma-separated list of device usernames. | Optional | 
| device_id | A comma-separated list of device IDs.  | Optional | 
| device_os | A comma-separated list of device operating systems. | Optional | 
| process_sha256 | A comma-separated list of process SHA256s. | Optional | 
| policy_name | A comma-separated list of policy names.  | Optional | 
| reputation | A comma-separated list of alert reputations.  | Optional | 
| alert_type | A comma-separated list of alert types. | Optional | 
| device_name | A comma-separated list of device names. Examples- "C:\\Users\\exmaple_user\\Example" or  "/home/example_user/Example". | Optional | 
| process_name | A comma-separated list of process names. Examples- "C:\\Users\\exmaple_user\\Example" or  "/home/example_user/Example". | Optional | 
```

| sort_field | Field by which to sort the results. Possible values are: first_event_timestamp, last_event_timestamp. Default is first_event_timestamp. | Optional | 
| sort_order | How to order the results. Can be "ASC" (ascending) or "DESC" (descending). Possible values are: ASC, DESC. Default is DESC. | Optional | 
| limit | The maximum number of results to return. Default is 10. | Optional | 
| start_time | Alert start time. | Optional | 
| end_time | Alert end time. | Optional | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| CarbonBlackEEDR.Alert.threat_id | String | Threat ID. | 
| CarbonBlackEEDR.Alert.first_event_timestamp | Date | First event time. | 
| CarbonBlackEEDR.Alert.target_value | String | Alert target value. | 
| CarbonBlackEEDR.Alert.reason | String | Alert reason. | 
| CarbonBlackEEDR.Alert.org_key | String | Organization key. | 
| CarbonBlackEEDR.Alert.device_id | String | Device ID. | 
| CarbonBlackEEDR.Alert.report_id | String | Report ID. | 
| CarbonBlackEEDR.Alert.watchlists.id | String | Watchlist ID. | 
| CarbonBlackEEDR.Alert.watchlists.name | String | Watchlist name. | 

> | CarbonBlackEEDR.Alert.device_os_version | String | Device OS version. | 

Reviewer: ```suggestion
| CarbonBlackEEDR.Alert.device_os_version | String | Device operating system version. | 
```

| CarbonBlackEEDR.Alert.threat_cause_threat_category | String | Threat cause threat category. | 
| CarbonBlackEEDR.Alert.policy_id | String | Policy ID. | 
| CarbonBlackEEDR.Alert.threat_indicators.process_name | String | Threat indicator - process name. | 
| CarbonBlackEEDR.Alert.threat_indicators.sha256 | String | Indicator SHA256 hash. | 
| CarbonBlackEEDR.Alert.threat_cause_actor_sha256 | String | Threat cause actor SHA256. | 

> | CarbonBlackEEDR.Alert.device_os | String | Device OS. | 

Reviewer: ```suggestion
| CarbonBlackEEDR.Alert.device_os | String | Device operating system. | 
```
